components/survey_monkey/survey_monkey.app.mjs

```diff
@@ -1,11 +1,150 @@
+import { axios } from "@pipedream/platform";
+import { v4 as uuid } from "uuid";
+import constants from "./common/constants.mjs";
+
 export default {
   type: "app",
   app: "survey_monkey",
-  propDefinitions: {},
+  propDefinitions: {
+    survey: {
+      type: "string",
+      label: "Survey",
+      description: "Survey where the action will be performed.",
+      async options() {
+        const { data } = await this.getSurveys();
+
+        return data.map((survey) => ({
+          label: survey.title,
+          value: survey.id,
+        }));
+      },
+    },
+    objectType: {
+      type: "string",
+      label: "Object type",
+      description: `Object type to filter events by: survey or collector. 
+        NOTE: Setting object_type to collector and event_type to collector_created will result in a 400 error. 
+        Object type should not be provided for survey_created webhook`,
+      options: [
+        "survey",
+        "collector",
+      ],
+      optional: false,
+      default: "",
+      reloadProps: true,
```

Comment:
It's recommended to specify `reloadProps` in the component instead of the app file. You can remove it from here and add it in `custom-webhook-events`. This also applies to the `eventType` prop.
```
    objectType: {
      propDefinition: [
        surveyMonkey,
        "objectType",
      ],
      reloadProps: true,
    },
```
```suggestion
```